ci/tests/vmachine/vmachine_test.py

```diff
@@ -114,3 +115,92 @@ def vms_with_fio_test():
                 counter = 0
         vpool = GeneralVPool.get_vpool_by_name(vpool_name=vpool_name)
         assert len(vpool.vdisks) == 0, "Still some disks left on the vpool after waiting {0} seconds: {1}".format(timeout, vpool.vdisks_guids)
+
+    @staticmethod
+    def check_scrubbing_test():
+        """
+        Check scrubbing of vdisks test
+        """
+        issues_found = ""
+        nr_of_disks = 3
+        vpool_name = General.get_config().get('vpool', 'name')
+        vpool = GeneralVPool.get_vpool_by_name(vpool_name=vpool_name)
+        assert vpool, "No vpool found where one was expected"
+
+        for disk_number in range(nr_of_disks):
+            disk_name = "scrubdisk-{0}".format(disk_number)
+            GeneralVMachine.logger.info("Starting RAW disk creation")
+            template_folder = GeneralVMachine.template_target_folder
+            image_name = GeneralVMachine.template_image
+            out, err, _ = General.execute_command('qemu-img convert -O raw {0}{1} /mnt/{2}/{3}.raw'.format(template_folder, image_name, vpool_name, disk_name))
+            if err:
+                GeneralVMachine.logger.error("Error while creating raw disk: {0}".format(err))
+
+        for vm_number in range(nr_of_disks):
+            machine_name = "machine-{0}".format(vm_number)
+            disk_name = "scrubdisk-{0}".format(vm_number)
+            GeneralVMachine.logger.info("Starting vmachine creation from RAW disk")
+            out, err, _ = General.execute_command('virt-install --connect qemu:///system -n {0} -r 512 --disk /mnt/{1}/{2}.raw,'
+                                                  'device=disk --noautoconsole --graphics vnc,listen=0.0.0.0 --vcpus=1 --network network=default,mac=RANDOM,'
+                                                  'model=e1000 --import'.format(machine_name, vpool_name, disk_name))
+            if err:
+                GeneralVMachine.logger.error("Error while creating vmachine: {0}".format(err))
+
+        # Waiting for 3 minute of FIO activity on vmachine
+        time.sleep(180)
```

Comment:
Please verify if virt-install already launches the vm too ... no need to start / stop / start them again
We should keep the time each test runs as short as possible ...